### PR TITLE
Create Revamp Chassis and Motor Signoff.md

### PR DESCRIPTION
**Pull request by JoshuaEgwuatu**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
The changes to the power subsystem should be handled separately. Once a design is in place for the changes that are needed for the movement.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Function of the subsystem:
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md:1]: Be brief. There's no reason to write lengthy paragraphs without having a lot to say.

**Review by JesseTNRoberts — COMMENTED** [review]
You aren't allowed to change the tread. 

These constraints aren't precise enough. For instance, what is the specific requirement of the first constraint? There isn't one because it has no numbers. Constraints must be precise enough to permit you to design and analyze the system.

Was this schematic copied from the previous team? If so, it must be cited. Further, the modifications need to be made clear.

This still doesn't meet the requirements for the order of the signoff document.

Why is there a battery in this signoff? This has nothing to do with a chassis or motor.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+This subsystem is designed to enhance the mechanical features of the movement and power subsystems of the Crawlspace Inspection Robot, enabling it to ascend a 30-degree incline and navigate through a wider range of crawlspace environments by boosting its power output.
+
+# Specifications and Constraints:
+
+1. The Robot’s weight distribution and electrical components shall be improved with better DC motors and lower center of gravity. The Robot’s weight distribution and electrical components shall be improved by using more efficient and powerful DC motors, we should lower the robot’s center of gravity, ensuring it remains (put here the value of the maximum height of the robot) from the ground.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: This isn't a constraint.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Function of the subsystem:
+
+This subsystem is designed to enhance the mechanical features of the movement and power subsystems of the Crawlspace Inspection Robot, enabling it to ascend a 30-degree incline and navigate through a wider range of crawlspace environments by boosting its power output.
+
+# Specifications and Constraints:
+
+1. The Robot’s weight distribution and electrical components shall be improved with better DC motors and lower center of gravity. The Robot’s weight distribution and electrical components shall be improved by using more efficient and powerful DC motors, we should lower the robot’s center of gravity, ensuring it remains (put here the value of the maximum height of the robot) from the ground. 
+2. The device shall have rubber treading for more friction on inclined surfaces. The robot shall have rubber treads with a minimum coefficient of friction of 0.8 or higher to ensure increased traction on inclined surfaces (0.8 or higher is often considered a reasonable benchmark for rubber treads on inclined surfaces).
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: This has been said more than once I believe. We won't be changing the treads.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+3. The Crawlspace Robot shall be significantly heavier than the original model.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: Not a constraint.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+4. Power consumption shall suffice using both batteries for overall output by supplying 360W to the device.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: How is this known?

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+5. Power subsystem shall supply more power than the current model by adding another battery in parallel to provide 14 A to the electrical system without increasing the voltage.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: The name of this is chassis and motor. Why is the power supply being discussed? No additional batteries are allowed. The system will work off one battery.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+6. The DC Motors selected shall help the Crawl Space Robot move easier on an incline with the motors that increase its torque with its 90:1 gear ratio and 38 kg⋅cm stall torque.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: I am not clear on what this is intended to say.

File: Movement/Signoffs/Revamp Chassis and Motor Signoff.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+7. For the power Subsystem, the motors can run off the main power lines of 12V DC and use the buck converter to power the microcontroller, the Raspberry Pi Model 4, to command the sensors and motor drivers.
```
Comment (JesseTNRoberts) [inline Movement/Signoffs/Revamp Chassis and Motor Signoff.md]: Not a constraint.

**Review by JesseTNRoberts — COMMENTED** [review]
Until the constraints are correct, I won't look at any design or analysis. The constraints guide the design. Without a design that is based on constraints, there is nothing to analyze.

**Review by JesseTNRoberts — COMMENTED** [review]
I don't understand why a battery is listed here.

**Comment by fnalheji42** [comment]
I removed the battery from the BOM. Sorry for the oversight.

**Review by JesseTNRoberts — APPROVED** [review]
Let's talk about this today in class. But it is approved.

**Pull request merged**